installation/resources/installer-cr-kyma-lite-compass.yaml

```diff
@@ -1,7 +1,10 @@
+apiVersion: "installer.kyma-project.io/v1alpha1"
 kind: Installation
```

Comment:
I would name this file differently. The current name suggests that it contains kyma lite, but its content is different than kyma-lite CR known from kyma documentation.

Reply:
I didn't even know there is kyma lite mentioned in docs, I'll try to come up with a better name :)